dir.props

```diff
@@ -546,6 +498,7 @@
     <SkipTests Condition="'$(SkipTests)'=='' and ('$(OsEnvironment)'=='Windows_NT' and '$(TargetsWindows)'!='true' and '$(OSGroup)'!='AnyOS')">true</SkipTests>
   </PropertyGroup>
 
-  <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'=='Unix' and Exists('$(RoslynPropsFile)')" />
-
+  <!-- Use Roslyn Compilers to build -->
+  <Import Project="$(RoslynPropsFile)" Condition="'$(OSEnvironment)'!='Unix' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
```

Comment:
I'm good not blocking this PR but we should try and clean this up some more. 

As for other repo's using BuildTools right now it is a little too difficult for them to start using BuildTools successfully because of all the hidden properties and imports like these that they don't know they need. Part of the work that @markwilkie is having @crummel  looking is to remove as much of the boilerplate as possible to allow repo's to consume BuildTools much easier. So this request is more along those lines.